### PR TITLE
Add GitHub footer, super basic contributing guide

### PR DESCRIPTION
**Pull request by CloCkWeRX**
#24 

I promise I'll stop firing off PR's soon :)

### CODE REVIEW

**Comment by alexandre-mbm** [comment]
No stop!

We are seeing the benefits of internationalization...

**Comment by CloCkWeRX** [comment]
Yeah I'll sprinkle in the i18n stuff here in a sec, hold off on merging

**Comment by CloCkWeRX** [comment]
Should be good to go; though more than happy if there's a better translation available!

File: locales/pt-BR/translation.json
Comment (alexandre-mbm) [inline locales/pt-BR/translation.json]: _Código-fonte disponível no_

**Review by alexandre-mbm — CHANGES_REQUESTED** [review]

File: locales/pt-BR/translation.json
Comment (CloCkWeRX) [reply locales/pt-BR/translation.json]: Thanks!

**Pull request merged**
